dev/PushNotifications/PushNotificationsLongRunningTask/platform.cpp

```diff
@@ -120,9 +113,7 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationsLongRunningPlatformImpl::RegisterF
     auto lock = m_lock.lock_exclusive();
     THROW_HR_IF(WPN_E_PLATFORM_UNAVAILABLE, m_shutdown);
 
-    const std::wstring appId = GetAppIdentifier(processName);
-
-    m_notificationListenerManager.AddListener(appId, processName);
+    const std::wstring appId{ GetAppIdentifier(processName) };
```

Comment:
If there is no appId for such processName, then we are going to add a sink anyway. I think we should throw with ERROR_NOT_FOUND. Thoughts?

Also, are there any SDK scenarios where we first do something else in the LRP than registering the app? i.e. register foreground activator for App1, then registering App1?

Reply:
We decided as a team to return S_OK on adding foreground sinks and toast registratrions. Remove will not fail if the appId doesn't exist.

To my knowledge there is no other SDK scenarios that could break this.